examples/equilibrium_temperature_co2.jl

```diff
@@ -0,0 +1,30 @@
+using Klimakoffer
+using Printf
+
+NT = 48 # Number of time-steps per year
+
+mesh = Mesh()
+
+model = Model(mesh, NT)
+
+discretization = Discretization(mesh, model, NT)
+
+year_start = 1950
+year_end = 2020
+year_delta = year_end-year_start+1
```

Comment:
```suggestion
year_delta = year_end - year_start + 1
```
For consistency :sunglasses: